atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/impl/ranges/DbKvsGetRanges.java

```diff
@@ -330,7 +335,14 @@ private String getSimpleRowSelectOneQueryOracle(
     }
 
     private String getPrefixedTableName(TableReference tableRef) {
-        return new PrefixedTableNames(config, new ConnectionSupplier(connectionSupplier)).get(tableRef);
+        switch (dbType) {
```

Comment:
`oracleTableNameGetter` was only used here to make a `prefixedTableNames` that understood oracle table names, so instead I created `prefixedTableNames` at the higher level inside the single `DbType` switch